__tests__/app.test.js

```diff
@@ -34,7 +34,48 @@ describe("/api/categories", () => {
         });
       });
   });
-  it("GET 404: Should respond with a 404 error if the url is invalid ", () => {
+});
+
+describe("/api/reviews/:review_id", () => {
+  it(`'GET 200: Should return a review object, which should have the following properties: 
+  review_id'title','review_body','designer','review_img_url','votes','category','owner','created_at'`, () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body: { review } }) => {
+        expect(review).toMatchObject({
+          review_id: expect.any(Number),
```

Comment:
We have access to the data so instead of expecting any data we can be really specific and hard code the response here and really make sure we are getting the correct data back